.vscode/launch.json

```diff
@@ -7,15 +7,16 @@
         "name": "Azure CLI Debug (Integrated Console)",
         "type": "python",
         "request": "launch",
-        "pythonPath": "${config:python.pythonPath}",
+        "pythonPath": "${config:python.interpreterPath}",
```

Comment:
this may be written by vscode, but better to user pythonPath which is set in the setting.json file. Let's check the usage of interpreterPath later to figure out the right way.

Reply:
OK